### DISCUSSION TITLE
[NHẬN TASK]

### DISCUSSION THREAD

**Discussion by nguyenxuanquang6789** [Runway-Club/Runway-CV-Builder#36] [Task Delivery]
Tôi muốn làm task giao diện Landing page

**Comment by nguyentantrung2000** [comment]
Chào bạn sau khi xem xét thì task này vẫn chưa có người làm nên bạn sẽ làm task này nhé.
Thông tin của task:
-Là đầu tiên khi người dùng vào sẽ thấy.
Yêu cầu:
Giao diện hợp lý , phù hợp đúng theo bố cục figma
Deadline 11:59PM 2/8/2021

**Reply by nguyenxuanquang6789** [reply]
Mình đồng ý !
